### DISCUSSION TITLE
Cesium Viewerport Render

### DISCUSSION THREAD

**Discussion by KorecGroup-AliJafri** [zouyaoji/vue-cesium#559] [Q&A]
Hi, Is there a way to pause viewer rendering till a all tilesets are loaded? I am loading tilesets in a vue-cesium component and have come across a situation where I may have to load 40 tilesets and the client prefers I avoid the viewport update referesh as each tileset loads. Below is a snippet of the component. 
```vue
<template v-if="asset.type === 'Tileset'">
            <vc-primitive-tileset
              :url="asset.url"
              :asset-id="asset.cesiumId"
              :visibility="tilesetVisibility[asset.id]"
              :maximum-screen-space-error="
                asset.maximumScreenSpaceError ? asset.maximumScreenSpaceError : 12
              "
              :dynamic-screen-space-error="
                asset.dynamicScreenSpaceError ? asset.dynamicScreenSpaceError : true
              "
              :model-matrix="modelMatrix"
              @ready="onReady"
            ></vc-primitive-tileset>
          </template>
```

**Comment by zouyaoji** [comment]
Try changing `scene.requestRenderMode` to see if it works?

![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/1473bdc8-0118-43be-8be2-89de1d1c76c5)

**Reply by KorecGroup-AliJafri** [reply]
Although it has shown marked improvement in performance but it still refreshes after each tileset load. Recorded a video to show what's happening with three tilesets loaded in the viewer. Main idea (if it's at all possible) is to pause the rendering till all tilesets load then viewer to show loaded tilesets in one go. They are currently being loaded in by reading a json config file and adding tilesets based on set parameters using a loop. Any further ideas would be greatly appreciated. Thanks

https://github.com/zouyaoji/vue-cesium/assets/112545008/acbae444-4541-470b-bd95-30160adb93b1
